core/blockchain.go

```diff
@@ -62,6 +63,12 @@ const (
 	maxTimeFutureBlocks = 30
 	badBlockLimit       = 10
 	triesInMemory       = 128
+	shardCacheLimit     = 2
+
+	// BlocksPerEpoch is the number of blocks in one epoch
+	// currently set to small number for testing
+	// in future, this need to be adjusted dynamically instead of constant
+	BlocksPerEpoch = 5
 
 	// BlockChainVersion ensures that an incompatible database forces a resync from scratch.
```

Comment:
We should make this a global config param, along with number of shards.

Reply:
In future, we should use a function to dynamically adjust the epoch size. It's to me more like a code level security parameter instead of a configuration parameter.